db/migrate/20211028232254_create_posts.rb

```diff
@@ -3,8 +3,8 @@ def change
     create_table :posts do |t|
       t.string :title
       t.text :text
-      t.integer :commentsCounter
-      t.integer :likesCounter
+      t.integer :comments_counter
+      t.integer :likes_counter
```

Comment:
- [ ] [optional] It is highly recommended to not edit migration files, you should rather create another migration file in order to make the changes you want, so that your db will stay in a consistent state, otherwise you might get difficult situation in production